.gitignore

```diff
@@ -1,49 +1,9 @@
-# dependencies
-node_modules
-.pnp
-.pnp.js
-
-# testing
-coverage
-
-# next.js
-.next/
-out/
-build/
-.swc
-
 # misc
 .DS_Store
+.*.swp
 .idea/
 *.pem
 .pg
 
-# debug
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-.pnpm-debug.log*
-
-# local env files
```

Comment:
@robinbb should we keep ignoring .env and related files? If we don't want to list them all can we .gitiignore `.env.*`? Any reason to remove them from .gitignore?

Reply:
They are still ignored from within the 'web' directory, via its .gitignore. file. Outside of the 'web' directory, these files would be unexpected.